File: src/deepLearning/char-rnn-tensorflow-master/predictClass/x.py
#! /usr/bin/python 
#encoding:gb18030
############################################
#
# Author: 
# E-Mail:@sogou-inc.com
# Create time: 2016 12月 21 17时31分27秒
# version 1.0
#
############################################


import sys,os,re

for line in sys.stdin:
  line = line.strip()
  if line.find("\u") != -1:
    line = line[1:]
    print line.encode("gbk")

File: src/deepLearning/char-rnn-tensorflow-master/x.pl
#! /usr/bin/perl
use Encode;
############################################
#
# Author: 
# Create time: 2016 12ÔÂ 21 17Ê±33·Ö12Ãë
# E-Mail: @sogou-inc.com
# version 1.0
#
############################################
while(my $line = <>)
{
  chomp $line;
  print decode("gbk", $line)."\n";
  next;
  if($line =~ /^V|^sV/)
  {
    $line =~ s/^sV//;
    $line =~ s/^V//;
    print encode("gbk",$line)."\n";
  }
}
